pkg/server/api_test.go

```diff
@@ -29,6 +29,8 @@ type scenario struct {
 	checkResponse checkResponse
 }
 
+const expectedContentLength int = 15
```

Comment:
Where did this change originate from?

Reply:
the marshaled data is much smaller now due to leaving out all the empty/nulled fields - it's a nice side benefit of using this parser. As the number (114 before, 15 now) is specified a few times, I introdued this constant.